src/mutate/index.js

```diff
@@ -147,6 +143,19 @@ const decorateDeleteBase = args => {
   );
 };
 
+const createOnSubmitHandler = (props, schema) => (data, validate) => {
+  let validData = data;
```

Comment:
make a copy of data so you don't mutate it directy

Reply:
validData = JSON.parse(JSON.stringify(data))

Reply:
validData is never mutated?